packages/core/cache/nuxt/index.js

```diff
@@ -1,93 +1,111 @@
 /* eslint-disable */
-import path from 'path';
-import { createMiddleware } from '@vue-storefront/core/server';
+import { isAbsolute, resolve } from 'path';
 import { Logger } from '@vue-storefront/core';
 
-function rendererFactory (renderFn) {
+/**
+ * Adds endpoint to invalidate cache
+ */
+function createInvalidationEndpoint (driver, options) {
+  if (!options) {
+    return;
+  }
+
+  this.addServerMiddleware({
+    path: options.endpoint,
+    handler: async (request, response) => {
+      try {
+        // Remove leading slash and get URL params
+        const params = new URLSearchParams(request.url.replace(/^\//, ''));
+        const tags = params.get('tags').split(',');
```

Comment:
I wonder if we maybe should support some special value like `*` in VSF1 which would purge whole cache (e.g. after deploy). Wdyt?

Reply:
Added